controllers/tf_controller_runner.go

```diff
@@ -462,3 +463,7 @@ func (r *TerraformReconciler) reconcileRunnerSecret(ctx context.Context, terrafo
 
 	return result.Secret, nil
 }
+
+func runnerPodInstance(revision string) string {
+	return fmt.Sprintf("tf-runner-%s", strings.Split(revision, ":")[1][0:8])
+}
```

Comment:
Please add some checks to make sure that we're not breaking the controller when a weird value being passed thru this function.
```suggestion
func runnerPodInstance(revision string) (string, error) {
        // please add some checks here
	return fmt.Sprintf("tf-runner-%s", strings.Split(revision, ":")[1][0:8])
}
```